### ISSUE TITLE
Remove using namespace from headers

### ISSUE THREAD

**Issue by rwj11** [ScottishCovidResponse/SCRCIssueTracking#585]
Having `using namespace std;` in header files goes against C++ coding guidelines

**Comment by rwj11** [comment]
See https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rs-using-directive

**Comment by ianhinder** [comment]
Yes - this should be fixed, but it's low priority, as the code is using it ubiquitously and it's not intended to include its header files from other projects.
